example/lib/ide/ide.dart

```diff
@@ -315,7 +346,9 @@ class BottomBar extends StatelessWidget {
             const SizedBox(width: 8),
             // This is here to experiment with the LspClient
             GestureDetector(
-              onTap: _renameFile,
+              onTap: () {
+                _renameFile(context);
+              },
```

Comment:
Can we have the rename be invoked by pressing ENTER on a file selected in the file tree instead of this button?

Reply:
Sure thing. I'll update it.